sample-apps/react/react-video-demo/src/components/LobbyPanel/LobbyPanel.tsx

```diff
@@ -65,18 +66,21 @@ export const LobbyPanel: FC<Props> = ({
 }) => {
   const [permissionsErrorComponent, setPermissionsErrorComponent] =
     useState<any>(() =>
-      permissionsEnabled ? (
-        <DisabledVideoPreview name={user.name} />
-      ) : (
+      permissionsEnabled === false || permissionsEnabled === undefined ? (
         <EnableBrowserSettings />
+      ) : (
+        <DisabledVideoPreview name={user.name} />
```

Comment:
I am not sure, if I understand, why there are 3 possible outcomes for a boolean value:

`permissionEnabled ? A : !permissionEnabled ? B : A`